src/snek5000/output/base.py

```diff
@@ -641,6 +642,11 @@ def write_snakemake_config(self, custom_env_vars=None, host=None):
         with open(path_configfile_simul, "w") as file:
             yaml.dump(config, file)
 
+        # Build Nek5000 if needed
+        nek5000 = _Nek5000Make()
+        if not nek5000.build(config):
+            raise RuntimeError("Nek5000 build failed.")
+
```

Comment:
Not the best place to do this, but here we have the up to date `config` dictionary.

Reply:
I moved this is post_init.